Scripts/developer_scripts/cgal_demo_copy_all_dlls_cygwin.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#use this script from inside the build directory of the Polyhedron demo
+
+declare config="Release"
+
+declare target_directory="CGAL_demo_with_dlls"
+if [[ ! -d "$target_directory" ]]
+then
+  mkdir $target_directory
+fi
+
+copy_dll()
+{
+  local dll_full_path="`cygpath --unix --absolute $1`"
+  echo "copy " $dll_full_path " to " $2
+  cp $dll_full_path $2
```

Comment:
```suggestion
  cp "$dll_full_path" "$2"
```

Reply:
@maxGimeno I suggest you commit the changes directly in Jane's branch. Anyway, you will test the script.